WPObjectMapper/NSObject+ObjectMap.m

```diff
@@ -39,35 +39,36 @@
 @protocol NSObjectMappable
 @optional
 + (NSDictionary *) mapping;
-+ (instancetype) objectWithDictionary: (NSDictionary *) dictionary;
++ (nullable instancetype) objectWithDictionary: (NSDictionary *) dictionary;
 @end
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
Why `NS_ASSUME_NONNULL_BEGIN` in the implementation file, and there is none in the header file, where it should be?

Reply:
You're free to open a PR and fix. Thanks.

Reply:
As long as I've not missed something...

Reply:
I think you're right. I just tried to get this done as quickly as possible as we had a client downstream getting crashes.

The problem stems from Ellipsis changes on the backend, and PostKit tried to create some dumb generic RainbowItem with a dictionary.